packages/backend/src/app.ts

```diff
@@ -395,15 +398,20 @@ export class App {
     )
 
     let tenantApiSignatureResult: TenantApiSignatureResult
-    if (this.config.env !== 'test') {
+    if (this.config.env === 'test') {
+      const tenantService = await this.container.use('tenantService')
```

Comment:
noticed this is returning an `any` type. Need to add `tenantService: Promise<TenantService>` to the `AppServices` interface in this file.